jwst/dark_current/dark_current_step.py

```diff
@@ -10,7 +10,7 @@ class DarkCurrentStep(Step):
     """
 
     spec = """
-        dark_output = output_file(default = None)
+        dark_output = output_file(default = None) # Dark model or averaged dark subtracted
```

Comment:
This should be OK, but just as a warning for the future, we have to be careful with what gets included in a comment field for the spec params. In the past we've noticed that including anything in parentheses in the comment, e.g. "# Switch for blee (Boolean)" causes the arg parser to think that's part of the definition for the param and immediately gets confused and chokes.